cmd/fortifyExecuteScan.go

```diff
@@ -400,19 +402,31 @@ func checkArtifactStatus(config fortifyExecuteScanOptions, sys fortify.System, p
 	return nil
 }
 
-func verifyScanResultsFinishedUploading(config fortifyExecuteScanOptions, sys fortify.System, projectVersionID int64, buildLabel string, filterSet *models.FilterSet, numInvokes int) error {
+func verifyScanResultsFinishedUploading(config fortifyExecuteScanOptions, sys fortify.System, projectVersionID int64, buildLabel string, filterSet *models.FilterSet) error {
```

Comment:
I think we can integrate these 2 statements in line 160 to avoid an extra function for this.

Reply:
Understood

Reply:
The extra function will be in the tests though, unless we want to add clutter to the existing tests.